assignment_6.py

```diff
@@ -0,0 +1,49 @@
+# Rearranging cars
+
+def rearrange_cars(cars,order):

```

Comment:
According to Google Python Style Guide:
- Follow standard typographic rules for the use of spaces around punctuation.
- Do use whitespace after a comma, semicolon, or colon except at the end of the line.

Could you add a space after cars please?